src/esp/core/ManagedContainer.h

```diff
@@ -14,14 +14,37 @@
 
 namespace esp {
 namespace core {
+
+/**
+ * @brief This enum describes how objects held in the ManagedConatainers are
+ * accessed.
+ */
+enum class ManagedContainerAccess {
+  /**
+   * @brief When an object is requested to be retrieved, a copy is made and
```

Comment:
FYI a `@brief` for enum values won't be displayed in the docs because &lt;complicated reasons&gt; -- do it without:

```suggestion
   * When an object is requested to be retrieved, a copy is made and
```

Same below.